src/jp/mcbe/fuyutsuki/Texter/text/FloatingText.php

```diff
@@ -125,10 +125,9 @@ public function asPackets(Player $player, SendType $type): array {
 					$uuid,
 					$this->replaceVariables($player),
 					$this->actorRuntimeId,
-					$this->actorRuntimeId,
 					"",
 					$this->position,
-					null,
+					$this->position,
```

Comment:
The sixth argument should be `null` for `?Vector3 $motion`

```suggestion
					null,
```